packages/markup-frontstage/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "@bentley/markup-frontstage-react",
+  "version": "1.0.0",
+  "description": "Markup Frontstage React",
+  "license": "MIT",
+  "main": "lib/markup-frontstage-react.js",
+  "typings": "lib/markup-frontstage-react",
+  "scripts": {
+    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize",
+    "build-prod": " tsc -p tsconfig.json 1>&2 && npm run copy:assets",
+    "clean": "rimraf lib",
+    "lint": "tslint -p . 1>&2",
```

Comment:
Looks like you're missing a `tslint.json` file or a config within the `package.json`

Reply:
All the other packages in the repo have the same issue.  Let's follow up in a separate PR to clean this up.